diktat-maven-plugin/src/main/kotlin/com/saveourtool/diktat/plugin/maven/DiktatBaseMojo.kt

```diff
@@ -160,7 +160,30 @@ abstract class DiktatBaseMojo : AbstractMojo() {
         return generateSequence(mavenProject) { it.parent }
             .map { it.basedir.toPath().resolve(diktatConfigFile) }
             .run {
-                firstOrNull { it.isRegularFile() } ?: first()
+                firstOrNull { it.isRegularFile() } ?: file
```

Comment:
does `file` really satisfy us, if it has relative path? (`?: file` branch)

Reply:
it's a some value, which is not corrected  -- we need to return something not null and then it will be handled as error later

Reply:
refactored to nullable